PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -651,13 +653,12 @@ void PATMuonProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetu
       bool isRun2016BCDEF = (272728 <= iEvent.run() && iEvent.run() <= 278808);
       muon.setSelectors(muon::makeSelectorBitset(muon, pv, isRun2016BCDEF));
     }
-    double miniIsoValue = -1;
+    float miniIsoValue = -1;
     if (computeMiniIso_){
       // MiniIsolation working points
 
       miniIsoValue = getRelMiniIsoPUCorrected(muon,*rho, effectiveAreaVec_);
 
-      muon.setSelector(reco::Muon::MiniIsoLoose,     miniIsoValue<0.40);
```

Comment:
on L737 you require ```muon.passed(reco::Muon::MiniIsoLoose)```  before setting any of the MVA flags.
It looks like this way by construction all MVA flags will be false.
Please check.